sphinx/builders/latex/transforms.py

```diff
@@ -38,7 +38,7 @@ def apply(self, **kwargs: Any) -> None:
 
 
 class SubstitutionDefinitionsRemover(SphinxPostTransform):
-    """Remove ``substitution_definition node from doctrees."""
+    """Remove `substitution_definition` node from doctrees."""
```

Comment:
Would you like to change this as a reference by any role? If so, it's better to use role name explicitly. If not, please use double back-ticks to create an inline literal string.

Reply:
I'm quite sorry, but I don't see what this particular comment refers to. It does not seems to be near the word "role".
In this case I just corrected what seemed to be a typo. I admit that I do not (yet) have in mind the meaning of the various enclosures. (Admittedly, given that this is a contribution to a documentation tool, it's slightly ironic. But in my defense, I'm new to Sphinx, my first use was last month for Python documentation)

Reply:
To be precise, a text surrounded by single backquotes is called an interpreted text in reStructuredText, and it's considered as a role. In Sphinx, the default role is `:any:`.
https://docutils.sourceforge.io/docs/ref/rst/roles.html

I guess you'd like mark-up this as a literal string. So it would be better to use double backquotes.
```suggestion
    """Remove ``substitution_definition`` node from doctrees."""
```